resources/admin.py

```diff
@@ -35,17 +38,57 @@ class BoatTypeAdmin(TranslatableAdmin):
     pass
 
 
-class HarborAdmin(CustomTranslatableAdmin, admin.OSMGeoAdmin):
-    list_display = ("name",)
+class HarborAdmin(CustomTranslatableAdmin, admin.OSMGeoAdmin, HarborManager):
     ordering = ("translations__name",)
+    readonly_fields = (
+        "max_width",
+        "max_length",
+        "max_depth",
+    )
+
+    def max_width(self, obj):
+        return obj.max_width
+
+    def max_length(self, obj):
+        return obj.max_length
+
+    def max_depth(self, obj):
+        return obj.max_depth
+
+    max_width.short_description = _("Maximum allowed width")
+    max_width.admin_order_field = _("max_width")
+
+    max_length.short_description = _("Maximum allowed length")
```

Comment:
nitpick, but could you put these right after the property declaration and without empty lines in between, i.e.
```python
def max_width(self, obj):
    return obj.max_width
max_width.short_description = _("Maximum allowed width")
max_width.admin_order_field = "max_width"

def max_length(self, obj):
    return obj.max_length
max_length.short_description = _("Maximum allowed length")
max_length.admin_order_field = "max_length"

def max_depth(self, obj):
    return obj.max_depth
max_depth.short_description = _("Maximum allowed depth")
max_depth.admin_order_field = "max_depth"
```
this is the convention that I've found in many projects and which seems easier to navigate 🙂

Reply:
Black automatically adds a whitespace in between tho.

Reply:
yeah, thought so 😅 that's ok 👍